server/util/grpc_server/grpc_server.go

```diff
@@ -146,6 +155,8 @@ func NewGRPCServer(env environment.Env, port int, credentialOption grpc.ServerOp
 
 	regServices(grpcServer, env)
 
+	channelzservice.RegisterChannelzServiceToServer(grpcServer)
```

Comment:
Does this mean the channelz rpc service will be exposed and accessible externally?

Reply:
hm good catch. I think I misunderstood how this works - this func is just exposing the same channelz service to each server, and the /channelz page contains results across all servers.

We don't have a "monitoring gRPC server" that this could be registered to, but would it make sense to register it to the internal gRPC server?

Reply:
registering to the internal gRPC server sounds good

Reply:
OK, done